kubernetes/samples/scripts/common/utility.sh

```diff
@@ -926,3 +926,26 @@ function checkService(){
  done
  echo "Service [$svc] found"
 }
+
+# Get pod name when pod available in a given namespace
+function getPodName(){
+
+ max=$((SECONDS + 120))
+
+ pod=$1
+ ns=$2
+
+ pname=""
+ while [ $SECONDS -le $max ] ; do
```

Comment:
Minor, but worth it: Please add "local " before each of max, pod, ns, and pname - in case they are globally used somewhere else.